tests/vars/qubes-rpc.yml

```diff
@@ -1,6 +1,8 @@
 - policy: qubes.ClipboardPaste
   starts_with: |-
     ### BEGIN securedrop-workstation ###
+    @tag:send-clipboard-to-sd @tag:sd-workstation ask
+    @tag:sd-workstation @tag:receive-clipboard-from-sd ask
```

Comment:
Good to have the tests updated. We also need a clean action, though: uninstalling should handle removal of these custom SDW-related tags. For example:

```
qvm-ls --tags sd-receive-logs --raw-list | xargs --no-run-if-empty -I {} qvm-tags {} del sd-receive-logs
```

for each custom tag.

Reply:
Ended up adding a Python script leveraging `qubesadmin` for this purpose.